basics_dev/coding-style.md

```diff
@@ -165,7 +165,7 @@ Security coding guidelines
        height = untrusted_conf.height;
     ~~~
 
--   Use another variables, without the `untrusted_` prefix to hold the sanitized values, as shown above.
```

Comment:
Sorry, stale page. Didn't see your 2nd message.

I'd say just "Use another variables" -> "Use other variables", keeping the original meaning exactly as-is.